src/stim/simulators/frame_simulator.h

```diff
@@ -97,6 +97,10 @@ struct FrameSimulator {
     void reset_all_and_run(const Circuit &circuit);
     void reset_all();
 
+    inline void invoke(uint8_t gate_id, const OperationData& data) {
```

Comment:
better name

Reply:
I renamed the function as `do_gate`. Hopefully that's clearer?